ghost/core/core/shared/config/defaults.json

```diff
@@ -102,6 +102,12 @@
             "maxWait": 43200000,
             "lifetime": 43200,
             "freeRetries": 8
+        },
+        "mentions_block": {
```

Comment:
RE: Naming - this block is specifically for incoming Webmentions - I think we should probably use the word webmentions rather than mentions in the naming.